pyro/poutine/__init__.py

```diff
@@ -150,16 +157,44 @@ def scale(fn, scale):
     scale factor, scale the score of all sample and observe sites in the
     function.
     """
-    m = ScaleMessenger(scale=scale)
-    # temporary change to preserve API
-    return m(fn) if callable(fn) else m
+    msngr = ScaleMessenger(scale=scale)
+    # XXX temporary compatibility fix
+    return msngr(fn) if callable(fn) else msngr
+
+
+def indep(fn=None, name=None, size=None, dim=None):
```

Comment:
I thought we deleted this since it was not user facing. Is there a reason to expose it?

Reply:
It makes it possible to replace the implementation in future